Renv/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# For finding latest versions of the base image see
+# https://github.com/SwissDataScienceCenter/renkulab-docker
+ARG RENKU_BASE_IMAGE=renku/renkulab-r:4.2.0-0.13.1
+FROM ${RENKU_BASE_IMAGE}
+
+# Uncomment and adapt if code is to be included in the image
+# COPY src /code/src
+
+# Uncomment and adapt if your R or python packages require extra linux (ubuntu) software
+# e.g. the following installs apt-utils and vim; each pkg on its own line, all lines
+# except for the last end with backslash '\' to continue the RUN line
+#
+USER root
+RUN apt-get -y update && \
+  apt-get clean && \
+  apt-get install -y --no-install-recommends \
+  apt-utils \
+    libncurses5-dev \
+    libncursesw5-dev \
+    parallel \
+    libgit2-dev \
+    tk-dev \
+    jq \
+    curl \
+    librsvg2-2 \
+    librsvg2-dev \
+    librsvg2-common
+
+# note jq and curl are used in renv_install.sh
+# librsvg2 libs are common dependencies of some R plotting libraries
+
+USER ${NB_USER}
+
+# install the R dependencies
+## make the renv install script and renv.lock file
+## available in the working dir and run the install
+COPY .renv_install.sh .
+COPY renv.lock .
+RUN bash .renv_install.sh
+## ensure renv lock is in the project directory
+COPY renv.lock /home/rstudio/renv.lock
+COPY install.R /tmp/
+RUN R -f /tmp/install.R
+
+# To apply a custom RStudio config uncomment the line below
```

Comment:
why not just add this by default?

Reply:
Mostly just because it is a change from the default behaviour in the other R session options like the basic and bioc ones. Wanted to keep it consistent with the behaviour people would expect from those by default. Persistent project specific RStudio config might be something it'd be nice to add to the other R templates but thought I'd start conservative with it as a off by default option here.

Reply:
sure that's fine! we can review this choice a bit later down the line or standardize with the others, as you say.